md/performance-tuning.md

```diff
@@ -406,13 +406,39 @@ Be sure to configure these appropriately.
 
 ## Connector time tracker
 
-It is now possible to track the duration of actions in a connector using a new time tracker. The tracker service tracks several connector lifecycle operations, and produces a CSV file.  
+It is now possible to track the duration of actions in a connector using a new time tracker. The tracker service tracks several connector lifecycle operations.  
 This service can impact performance so is disabled by default.  
-It is configured by editing `bonita-tenant-community-custom.properties` and `bonita-tenants-custom.xml`.  
-To activate connector time tracking: 
+It is configured by editing the following parameters in `bonita-tenant-community-custom.properties`.  
+
+```
+## Time tracker
+#bonita.tenant.timetracker.startTracking=false
+#bonita.tenant.timetracker.maxSize=1000
+#bonita.tenant.timetracker.flushIntervalInSeconds=30
+#bonita.tenant.timetracker.csv.activateAtStart=true
+#bonita.tenant.timetracker.csv.folder=$ {java.io.tmpdir}
+
+#bonita.tenant.timetracker.csv.separator=;
+#bonita.tenant.timetracker.memory.activateAtStart=false
+#bonita.tenant.timetracker.memory.maxSize=1000000
+```
+To activate connector time tracking:
+1. Uncomment all the previous lines except ```## Time tracker```.
+2. Change the value of `startTracking` from `false` to `true`.
+
+The other parameters can be left at their default value, left commented, or set to the desired value. What each of them does:
+
+1. `maxSize` maximum of records that will be saved by the time tracker before a flush.
+2. `flushIntervalInSeconds` the interval beetween two flushes on the timetracker thread.
```

Comment:
If flushIntervalInSeconds and maxSize are defined what happens? Is there a flush every flushIntervalInSecondsand AND everytime maxSize is reached before the interval complete?

Or those attributes should not be defined at the same time?